packages/auth/cloud/index.ts

```diff
@@ -0,0 +1,5 @@
+import { getTemplate } from "./cloudformation"
```

Comment:
In this case, you need to export `getTemplate`

```
export { getTemplate } from "./cloudformation"
```

Reply:
I tried to use the line above, but it's give me a error.